<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="GURU 99 Test SUITE" duration-ms="22387" started-at="2018-07-21T22:39:45Z" finished-at="2018-07-21T22:40:08Z">
    <groups>
    </groups>
    <test name="Logging In for the 100th Time" duration-ms="22387" started-at="2018-07-21T22:39:45Z" finished-at="2018-07-21T22:40:08Z">
      <class name="com.guru99.qa.testcases.LoginTestUsingListeners">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.guru99.qa.testcases.LoginTestUsingListeners@343f4d3d]" name="setUp" is-config="true" duration-ms="10153" started-at="2018-07-21T17:39:46Z" finished-at="2018-07-21T17:39:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="titleTest()[pri:1, instance:com.guru99.qa.testcases.LoginTestUsingListeners@343f4d3d]" name="titleTest" duration-ms="59" started-at="2018-07-21T17:39:56Z" finished-at="2018-07-21T17:39:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Guru is so cool] but found [Guru99 Bank Home Page]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Guru is so cool] but found [Guru99 Bank Home Page]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.guru99.qa.testcases.LoginTestUsingListeners.titleTest(LoginTestUsingListeners.java:33)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.guru99.qa.testcases.LoginTestUsingListeners@343f4d3d]" name="tearDown" is-config="true" duration-ms="686" started-at="2018-07-21T17:39:56Z" finished-at="2018-07-21T17:39:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.guru99.qa.testcases.LoginTestUsingListeners@343f4d3d]" name="setUp" is-config="true" duration-ms="8326" started-at="2018-07-21T17:39:57Z" finished-at="2018-07-21T17:40:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest()[pri:2, instance:com.guru99.qa.testcases.LoginTestUsingListeners@343f4d3d]" name="loginTest" duration-ms="1848" started-at="2018-07-21T17:40:05Z" finished-at="2018-07-21T17:40:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.guru99.qa.testcases.LoginTestUsingListeners@343f4d3d]" name="tearDown" is-config="true" duration-ms="752" started-at="2018-07-21T17:40:07Z" finished-at="2018-07-21T17:40:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.guru99.qa.testcases.LoginTestUsingListeners -->
    </test> <!-- Logging In for the 100th Time -->
  </suite> <!-- GURU 99 Test SUITE -->
</testng-results>
